MediaBrowser.Plugins.Anime/Configuration/configPage.html

```diff
@@ -43,13 +43,19 @@
                             <label for="chkMyAnimeList_API_Name">
                                 MyAnimeList Name. -not rly needed
                             </label>
-                            <input id="chkMyAnimeList_API_Name" name="chkMyAnimeList_API_Name" type="text" value="" disabled/>
+                            <input id="chkMyAnimeList_API_Name" name="chkMyAnimeList_API_Name" type="text" value="" disabled />
                         </li>
                         <li>
                             <label for="chkMyAnimeList_API_Pw">
                                 MyAnimeList Pw. -not rly needed
                             </label>
-                            <input id="chkMyAnimeList_API_Pw" name="chkMyAnimeList_API_Pw" type="password" value="" disabled/>
+                            <input id="chkMyAnimeList_API_Pw" name="chkMyAnimeList_API_Pw" type="password" value="" disabled />
```

Comment:
if we would not need passwords in the future, corresponding configuration field could be removed